src/components/DropdownMenu/DropdownMenu.test.tsx

```
@@ -530,3 +535,560 @@ describe('MenuContent direction and positioning', () => {
+      expect(screen.getByText('Ana Paula')).toBeInTheDocument();
+      expect(screen.getByText('ana@gmail.com')).toBeInTheDocument();
+    });
+  });
+
+  describe('ProfileMenuSection behavior', () => {
+    it('renders ProfileMenuSection with children', () => {
+      render(
+        <DropdownMenu open>
+          <ProfileMenuTrigger />
+          <MenuContent>
+            <ProfileMenuSection data-testid='ProfileMenuSection'>
```

ESLint

[error] Replace `'ProfileMenuSection'` with `"ProfileMenuSection"`

(prettier/prettier)